src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -260,6 +260,20 @@ kibana_vars=(
     xpack.rollup.enabled
     xpack.searchprofiler.enabled
     xpack.security.audit.enabled
+    xpack.security.audit.appender.type
+    xpack.security.audit.appender.layout.type
+    xpack.security.audit.appender.layout.highlight
+    xpack.security.audit.appender.layout.pattern
+    xpack.security.audit.appender.legacyLoggingConfig
+    xpack.security.audit.appender.fileName
+    xpack.security.audit.appender.policy.type
+    xpack.security.audit.appender.policy.interval
+    xpack.security.audit.appender.policy.modulate
+    xpack.security.audit.appender.policy.size
+    xpack.security.audit.appender.strategy.type
+    xpack.security.audit.appender.strategy.max
+    xpack.security.audit.appender.strategy.pattern
+    xpack.security.audit.ignore_filters
```

Comment:
The `ignore_filters` setting is an [array](https://www.elastic.co/guide/en/kibana/current/security-settings-kb.html#audit-logging-ignore-filters). Users would configure `.ignore_filters[].actions[]`, `.ignore_filters[].categories[]`, etc. But I think that we shouldn't specify `actions, `categories`, etc. in the Docker kibana vars. Is that correct?

Reply:
:+1: for now, open issue at https://github.com/elastic/kibana/issues/39595